packages/contracts-bedrock/contracts/test/MerkleTrie.t.sol

```diff
@@ -5,59 +5,56 @@ import { CommonTest } from "./CommonTest.t.sol";
 import { MerkleTrie } from "../libraries/trie/MerkleTrie.sol";
 
 contract MerkleTrie_Test is CommonTest {
+    function setUp() public {
+        _setUp();
+    }
+
     function test_get_validProofSucceeds1() external {
         bytes32 root = 0xd582f99275e227a1cf4284899e5ff06ee56da8859be71b553397c69151bc942f;
         bytes memory key = hex"6b6579326262";
         bytes memory val = hex"6176616c32";
         bytes[] memory proof = new bytes[](3);
-        proof[0] = hex"e68416b65793a03101b4447781f1e6c51ce76c709274fc80bd064f3a58ff981b6015348a826386";
-        proof[1] = hex"f84580a0582eed8dd051b823d13f8648cdcd08aa2d8dac239f458863c4620e8c4d605debca83206262856176616c32ca83206363856176616c3380808080808080808080808080";
+        proof[
+            0
+        ] = hex"e68416b65793a03101b4447781f1e6c51ce76c709274fc80bd064f3a58ff981b6015348a826386";
+        proof[
+            1
+        ] = hex"f84580a0582eed8dd051b823d13f8648cdcd08aa2d8dac239f458863c4620e8c4d605debca83206262856176616c32ca83206363856176616c3380808080808080808080808080";
```

Comment:
Looks like a lot of formatter changes. Could we move these into another PR below this one?

Reply:
ditto

Reply:
I put it into #4319 already actually.